cgmes/cgmes-model/src/main/resources/CIM100.sparql

```diff
@@ -30,3 +30,63 @@ OPTIONAL { GRAPH ?grapBDTP {
 }}
 }
 
+# query: synchronousMachines
+SELECT *
+WHERE {
+{ GRAPH ?graph {
+    ?SynchronousMachine
+        a cim:SynchronousMachine ;
+        cim:IdentifiedObject.name ?name ;
+        cim:RotatingMachine.GeneratingUnit ?GeneratingUnit .
+    OPTIONAL {
+        ?SynchronousMachine cim:RotatingMachine.ratedS ?ratedS
+       }
+    ?GeneratingUnit
+        a ?generatingUnitType ;
+        cim:GeneratingUnit.minOperatingP ?minP ;
+        cim:GeneratingUnit.maxOperatingP ?maxP ;
+        cim:GeneratingUnit.nominalP ?initialP ;
```

Comment:
initialP attribute has been removed from EQ in CIM100: _This required attribute is deleted as there is a general agreement that it is not expected to be able to run power flow on the EQ alone. SSH is required to be able to execute power flow._ 

So previously a required attribute, now can be dropped entirely, as initial values should come from SSH. In the conversion, a default value of zero should be set if no data from SSH is read.